### PR TITLE
Add Travis CI file for automated tests

### PR DESCRIPTION
**Pull request by meisenzahl**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+
+language: node_js
```
Comment (GijsGoudzwaard) [inline .travis.yml]: Why Node.js? The used language for this application is Vala

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+node_js:
+  - 10/*
+
+sudo: required
+
+services:
+  - docker
```
Comment (GijsGoudzwaard) [inline .travis.yml]: Docker isn't used in this app

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - libstdc++-5-dev
```
Comment (GijsGoudzwaard) [inline .travis.yml]: Shouldn't the packages listed in the README be added here?
E.g.

```
granite
gtk+-3.0
glib-2.0
jpegoptim
optipng
```

File: .travis.yml (hunk 1 above)
Comment (meisenzahl) [reply .travis.yml]: Node.js is used to run a tool of the elementary project. The tool checks your app for compatibility.

File: .travis.yml (hunk 2 above)
Comment (meisenzahl) [reply .travis.yml]: Docker is used to run the elementary project's tool on Travis CI.

File: .travis.yml (hunk 3 above)
Comment (meisenzahl) [reply .travis.yml]: The tool automatically pulls the dependencies from your project.

**Comment by meisenzahl** [comment]
If you want to know more about it, have a look at https://github.com/elementary/houston/wiki/Continuous-Integration.

**Pull request merged**